### ISSUE TITLE
Error when updating to 3.0.0

### ISSUE THREAD

**Issue by JordyLangen** [braze-inc/braze-flutter-sdk#35]
I'm getting the following error:

```
The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (/.../ios/Pods/BrazeKit/BrazeKit.xcframework and /.../ios/Pods/BrazeLocation/BrazeLocation.xcframework)
```

Changing `use_frameworks!` to `use_frameworks! :linkage => :static` seems to be a workaround, would prefer to see a fix.

**Comment by hokstuff** [comment]
Hi @JordyLangen,

If you remove the `use_frameworks!` line altogether in your `Podfile`, does this also address your issue? This was a manual step done for the Runner example project ([shown here](https://github.com/braze-inc/braze-flutter-sdk/commit/59832865d94e6873df657aa516b83f2794495409#diff-4fb44f272217447e6b4b3580916d09d23b9055f6a217ad8b26269cfa6031203a)) and is something we will add as a migration step when updating to 3.0.0. Also, the `use_frameworks` line is not required when importing the Braze Swift SDK ([see native examples here](https://github.com/braze-inc/braze-swift-sdk/blob/main/Examples/Swift/Podfile)).

**Comment by Icety** [comment]
I created a fork of the project and added 
`s.static_framework = true`
on line 12 of ios/braze_plugin.podspec

This way there is no need to change the use_frameworks in the podFile

**Comment by hokstuff** [comment]
Hi @Icety,

Thanks for the suggestion. We will integrate the change directly in the podspec in the next release to save integrators from needing to do an additional migration step. When a new release is available, I'll respond to this thread.

In the meantime, please use the workarounds in this thread.

Thank you!

**Comment by ggirotto** [comment]
@Icety Can you link the fork you made? I tried to fork it with `s.static_framework = true` but I'm facing `Swift Compiler Error (Xcode): No such module 'Appboy_iOS_SDK'`.

Removing `use_frameworks` from Podfile conflicts with other Google dependencies, didn't work for me.

**Comment by hokstuff** [comment]
Hi @JordyLangen @Icety @ggirotto,

We just released the [Braze Flutter SDK version 3.0.1](https://github.com/braze-inc/braze-flutter-sdk/releases/tag/3.0.1) which addresses the issues in this thread related to importing the static framework in the podspec. After updating to that version, you will no longer need to do a manual step to remove `use_frameworks!` or to use `use_frameworks! :linkage => :static`.

Thanks!
